docs/developer/endpoints/components/community-rpc/endpoints.const.ts

```diff
@@ -0,0 +1,278 @@
+export const communityMainnetEndpointsList = [
```

Comment:
@Artur-Sg Are they all verified to be working (and serving archive when applicable)?

Reply:
We've checked their availability. Please ensure that the endpoints for 'Pathrock Network' and 'Node101' are working, as they currently appear to be unavailable.